src/BlazorStrap/Components/Tabs/BSTab.razor.cs

```diff
@@ -25,13 +25,14 @@ public abstract class BSTabBase : ComponentBase, IDisposable
         [Parameter] public string Class { get; set; }
         [Parameter] public RenderFragment ChildContent { get; set; }
         [Parameter] public string Name { get; set; }
+        [Parameter] public bool? SetSelected { get; set; }
```

Comment:
Prefer InitialSelection. People will get confused and think toggling this will change the tab after it loads.

Reply:
@MarkZither Can you change the variable name? We will merge once that is done.